src/games/galaxian/galaxian.py

```diff
@@ -1,6 +1,113 @@
 from src.games.game import *
+from src.controls.joystick import *
+from src.settings import GameLevel
+from src.games.galaxian.player import *
+from src.games.galaxian.enemy import *
+import pygame
+import sys
+import random
 
 
 class Galaxian(Game):
+    SPEEDS = {GameLevel.Easy: 10, GameLevel.Medium: 20, GameLevel.Hard: 40}
+
     def __init__(self, level, is_sound_on):
         Game.__init__(self, level, is_sound_on)
+        self._points = 0
+        self._lives = 3
+        self._enemies = pygame.sprite.Group()
+        self._projectiles = pygame.sprite.Group()
+        self._player = Player(x=LayoutRsc.GAME_AREA_WIDTH / 2,
+                              y=LayoutRsc.GAME_AREA_HEIGHT - Player.HEIGHT/2,
+                              projectiles=self._projectiles,
+                              is_sound_on=is_sound_on)
+        self.enemy_speed = Galaxian.SPEEDS[level]
+        self._add_row_of_enemies()
+        self._first_enemies_row_y = 0
+
+    def _add_row_of_enemies(self):
+        num_of_enemies = 7
+        enemies = []
+        special_idx = random.randrange(0, num_of_enemies)
```

Comment:
special_enemy_index would be better, special_idx looks like special index and is not talkative

Reply:
ok